src/components/structures/RoomView.tsx

```diff
@@ -423,21 +423,21 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
 
             const thread = initialEvent?.getThread();
             if (thread && !initialEvent?.isThreadRoot) {
-                dispatchShowThreadEvent(
-                    thread.rootEvent,
-                    initialEvent,
-                    RoomViewStore.isInitialEventHighlighted(),
-                );
+                // dispatchShowThreadEvent(
+                //     thread.rootEvent,
+                //     initialEvent,
+                //     RoomViewStore.isInitialEventHighlighted(),
+                // );
```

Comment:
How come this has been commented out? I believe this is needed when one follows a matrix.to link

Reply:
It is commented out so that I know there is still work to do. I wanted to ask you, if the rest of the logic can also be removed here.
It is removed because (I think) its purpose was to hide/show the thread when switching rooms. Since this is now done by the store, (the store just shows the correct panel for the new room you open) it was not necessary anymore.
But thinking about it again it might also be used, to show the thread panel when using a link to an event? Than this is still needed. Can you confirm that/ explain what it does? Maybe the correct thing here would be to not remove it but change it so that it does not interfere with what the right panel store wants to do by adding another condition.

Reply:
Yes, I believe it's needed to jump to a specific event in a thread. For example, find some event in a thread, go to message options, share, copy the message link. Now change rooms, and enter `/goto <message link>` in the composer. This should jump to the room, open the thread, and highlight the specific event.

Reply:
Thanks for explaingin how to force that situation (wasnt aware of `/goto`). So this code wont be commented after all. But changed.

Reply:
This now got changed again. and it seems everything behaves as intended now.